core/camera/src/main/java/com/google/jetpackcamera/core/camera/CameraXCameraUseCase.kt

```diff
@@ -526,65 +536,75 @@ constructor(
             )
                 == PackageManager.PERMISSION_GRANTED
             )
-        val captureTypeString =
-            when (captureMode) {
-                CaptureMode.MULTI_STREAM -> "MultiStream"
-                CaptureMode.SINGLE_STREAM -> "SingleStream"
-            }
-        val name = "JCA-recording-${Date()}-$captureTypeString.mp4"
-        val contentValues =
-            ContentValues().apply {
-                put(MediaStore.Video.Media.DISPLAY_NAME, name)
-            }
-        val mediaStoreOutput =
-            MediaStoreOutputOptions.Builder(
-                application.contentResolver,
-                MediaStore.Video.Media.EXTERNAL_CONTENT_URI
-            )
-                .setContentValues(contentValues)
-                .build()
 
+        val pendingRecord = if (!ignoreUri) {
+            val fileOutputOptions = FileOutputOptions.Builder(
+                File(videoCaptureUri!!.getPath())
```

Comment:
should we force unwrap here? is it possible at all that `videoCaptureUri` is null?

Reply:
The problems have been resolved.

null check at line 521. videoCaptureUri can't be null here